tests/helpers/list_files_from_local_data.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+
+def list_files_from_local_data(who_is_asking_file, dir_type):
+    dir_types = ["frontend", "output", "repo", "user"]
+    assert dir_type in dir_types, "Argument dir_type should be one of {0}".format(str(dir_types))
+
+    parent_dir = os.path.dirname(os.path.realpath(who_is_asking_file))
+
+    d = {
+        "frontend": "frontend-files",
+        "output": "snippets",
```

Comment:
Can we call `output` as `snippets` as well for consistency. We should also rename it in other files.

Reply:
done in upcoming commit